File: PuntoVentaDulces/PuntoVentaDulces/Dulce.cs
namespace Clases
{

    public class Dulce : Producto
    {
        private string tipo;

        public Dulce(string codigo, string nombre, double precio, int stock, string tipo)
            : base(codigo, nombre, precio, stock)
        {
            this.tipo = tipo;
        }

        public string GetTipo() { return tipo; }

        public override string ToString()
        {
           
            return base.ToString() + " | Tipo: " + tipo;
        }
    }
}

File: Java/PuntoVentaDulces/src/Clases/Dulce.java
package Clases;

public class Dulce extends Producto {
    private String tipo; 

    public Dulce(String codigo, String nombre, double precio, int stock, String tipo) {
        super(codigo, nombre, precio, stock); 
        this.tipo = tipo;
   }
    public String getTipo() { return tipo; }
    @Override
    public String toString() {
        return super.toString() + " | Tipo: " + tipo;
    }
}
